CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ## 🛑 Breaking changes 🛑
 
 - `prometheusexporter`: Automatically rename metrics with units to follow Prometheus naming convention (#8950)
+- `datadogexporter`: The `traces.span_name_remappings` setting now correctly refers to the OpenTelemetry key without any sort of normalization. (#9693)
```

Comment:
Can we detect when this happens and warn users about this in the logs in some way? We still can do breaking changes like this but it would be nice to give people a heads up.

Reply:
I moved this to the Bug Fixes section instead, because it's more like a bug. You would expect the key of the name to be replace to be the one that you set in code, not Datadog's interpretation of it, which could be confusing.

Reply:
Okay, I think it's fine to consider this a bug fix. Some users mentioned this on an issue, so I will mention this there as a heads up to them